_includes/footer.html

```diff
@@ -0,0 +1,3 @@
+<footer>
+  <span>&copy; 2011-2015 TJ Holowaychuk, Travis Jeffery, Christopher Hiller & Joshua Boy Nicolai Appelman. All rights reserved.  Licensed <a href="http://opensource.org/licenses/MIT" target="_blank">MIT</a></span>
```

Comment:
I've changed it to precisely that, but it doesn't come out looking all that good, see http://blog.jbnicolai.com/mocha/#more-information